src/routes/api/users.js

```diff
@@ -1,11 +1,17 @@
+import userController from '../../controllers/users.controller';
+import validateUser from '../../middlewares/ValidateUser';
+import isUserExist from '../../middlewares/findUser';
+
 const router = require('express').Router();
```

Comment:
Since we are having different routers in different files, this file shoud be `authRouter` while this variable should be called `authRouter` as well, instead of just router.